### ISSUE TITLE
need help in my plot

### ISSUE THREAD

**Issue by Goziephy** [gimli-org/gimli#619]
### Problem description
Need help with my plot

Please describe your issue here.
Good evening Prof., I sent you traveltime.TOM file the other day and you plotted it and share the code with me. Your plot appeals exactly as it is in reflexW software with scale 10 to 60 seconds on traveltime axis); But when i run the code with the file, my plot displaced the traveltime (y-axis)  having 0.01 to 0.06 scale which is different from yours. Please what can i do?
Hope to hear from you
### Your environment: Jupyter-notebook 

Please provide the output of `print(pygimli.Report())` here. If that does not
work, please give provide some additional information on your:

![image001](https://github.com/gimli-org/gimli/assets/89167590/52412430-4c0c-4c99-9ba4-9b86feafa0fd)
![20231121_154300](https://github.com/gimli-org/gimli/assets/89167590/44dae9a9-8d0b-412c-9708-b606deefd26f)

**Operating system**: Windows
**Python version**: 3.11.5
**pyGIMLi version**: Output of `print(pygimli.__version__):1.4.4
**Way of installation**: e.g. Conda package, manual compilation from source, etc.

### Steps to reproduce

Tell us how to reproduce this issue. Ideally, you could paste the code that produces the error:

```python
import pygimli as pg
...
```

### Expected behavior

Tell us what should happen or what you want to achieve.
what do have the scale with you on traveltime axis

### Actual behavior

Tell us what happens instead and/or provide the output of your script.

**Comment by halbmy** [comment]
Note that I added the line `data["t"] /= 1000` because I guess the traveltime in the file is in obviously ms and not s. This needs to be changed in the import routine (if the TOM file always contains ms) but until then you will have to multiply by yourself.

**Comment by makeabhishek** [comment]
what is `data[t]/=1000`  in line 5?

**Comment by halbmy** [comment]
the traveltimes are divided by 1000

**Comment by Goziephy** [comment]
Thank you all for replying, The code is now running fine, but the second step apparent velocity is not showing well. Attached is the screenshot of the code and the displace. Hope to get a response
![20231121_183559](https://github.com/gimli-org/gimli/assets/89167590/8dd5b857-09b5-43bd-b963-0e370cfed758)

**Comment by Goziephy** [comment]
> Note that I added the line `data["t"] /= 1000` because I guess the traveltime in the file is in obviously ms and not s. This needs to be changed in the import routine (if the TOM file always contains ms) but until then you will have to multiply by yourself.

Thank you prof., The code is now running fine, but the second step apparent velocity is not showing well. Attached is the screenshot of the code and the displace. Hope to get a response
![20231121_185138](https://github.com/gimli-org/gimli/assets/89167590/7e471bc7-c304-4dce-8848-c38998989a43)

**Comment by halbmy** [comment]
As the output says, you seem to have zero traveltime leading to infinite apparent velocity. Remove those data with `data.remove(data["t"] <= 0]`.

**Comment by Goziephy** [comment]
> As the output says, you seem to have zero traveltime leading to infinite apparent velocity. Remove those data with `data.remove(data["t"] <= 0])`.

Thanks prof., the output from the code displaced this; as in the attached file but my apparent velocity is showing range 0.26. to 5.26. I don't know if the range is okay or needs to be multiplied. moreover there are some empty space in the model which i don't know if is okay too.
![20231121_221209](https://github.com/gimli-org/gimli/assets/89167590/57b52e5a-b336-4610-9ea0-355c1cde2950)

**Comment by halbmy** [comment]
I am pretty sure you DO need to multiply your data with 0.001 as discussed before. Which wave is so slow to have a traveltime of 60 seconds on a distance of 100m?? The lowest values would then be around 300m/s (the speed in air) and the highest at 5000 m/s (typical values for hard rock). The missing values are probably the ones with zero traveltime.

And please, don't send smartphone photos. You already have a notebook and can copy the output into the issue.

**Comment by Goziephy** [comment]
> I am pretty sure you DO need to multiply your data with 0.001 as discussed before. Which wave is so slow to have a traveltime of 60 seconds on a distance of 100m?? The lowest values would then be around 300m/s (the speed in air) and the highest at 5000 m/s (typical values for hard rock). The missing values are probably the ones with zero travel-time.
> 
> And please, don't send smartphone photos. You already have a notebook and can copy the output into the issue.

Thank you prof., I multiplied 0.001 in the code and my velocity range is still showing 0.26 to 5.26. Attached is my output and script.
![Figure_1](https://github.com/gimli-org/gimli/assets/89167590/92fb59d6-0f51-424f-b1a7-aec6d81d3c46)
![Figure_2](https://github.com/gimli-org/gimli/assets/89167590/fe1c545c-ef3e-4111-9182-19acc716e08c)
[SRTD.txt](https://github.com/gimli-org/gimli/files/13440897/SRTD.txt)

**Comment by Goziephy** [comment]
> I am pretty sure you DO need to multiply your data with 0.001 as discussed before. Which wave is so slow to have a traveltime of 60 seconds on a distance of 100m?? The lowest values would then be around 300m/s (the speed in air) and the highest at 5000 m/s (typical values for hard rock). The missing values are probably the ones with zero traveltime.
> 
> And please, don't send smartphone photos. You already have a notebook and can copy the output into the issue.

Prof. the code is running fine if i run it in anaconda prompt but in jupyter-notebook is displacing 0.26 to 5.27 m/s for velocity

**Comment by halbmy** [comment]
The plots look correct. But if you run the same code on the command line and in a notebook (using the same pygimli in the same environment), the results should be the same.

**Comment by Goziephy** [comment]
> The plots look correct. But if you run the same code on the command line and in a notebook (using the same pygimli in the same environment), the results should be the same.

Yes prof. is the same now IF I JOIN all the code together but i run this code below for fit between measured (crosses) and modeled (lines) travel-times and i am having a empty displace cells saying ( Line2D.set() got an unexpected keyword argument 'orientation'). Attached is the code, output and error message.

mgr.invert(secNodes=3, paraMaxCellSize=5.0,
           zWeight=0.2, vTop=500, vBottom=5000, verbose=1)
mgr.showFit(firstPicks=True) 

[SRTT.txt](https://github.com/gimli-org/gimli/files/13448779/SRTT.txt)
![Figure_3](https://github.com/gimli-org/gimli/assets/89167590/7edf3d5c-6a73-43ad-9e63-38d11315a07e)

**Comment by halbmy** [comment]
Yes, that's definitely an error that need to be corrected. Thank you for point out.

**Comment by Goziephy** [comment]
> Yes, that's definitely an error that need to be corrected. Thank you for point out.

Good evening Prof., 
Prof., the code is still not working yet. still showing as before

**Comment by halbmy** [comment]
`TravelTimeManager.showFit()` (`firstPicks` is True by default) shows the measured data as crosses along with the forward response as lines like demonstrated in the traveltime field data example
![image](https://github.com/gimli-org/gimli/assets/7136401/ec1c5581-e1a6-4a44-b777-2b719fc9ec77)

There was apparently some bug in one of the previous versions which was fixed with the last version v1.4.6.
